src/components/DatabaseList/index.js

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import Header from "../Header";
+import InformationBar from "../InformationBar";
+// import SideNav from "../SideNav";
+import Status from "../Status";
+import Spinner from "../Spinner";
+
+var databases = [
+    {
+        "url": "africakla1.cranecloud.io",
+        "name": "AirQo-users",
+        "size": 0.1,
+        "status": true,
+        "time": "28 seconds ago"
+    },
+    {
+        "url": "africa2.cranecloud.io",
+        "name": "AirQo-devices",
+        "size": 32,
+        "status": false,
+        "time": "4 days ago"
+    },
+    {
+        "url": "defaultafrica.cranecloud.io",
+        "name": "VotingApp-db",
+        "size": 5,
+        "status": true,
+        "time": "5 minutes ago"
+    },
+];
+
+class DatabaseList extends React.Component {
+
+  render() {
+    const isRetrieving = false;
+    const isFetched = true;
+    return (
+      <div className="MainPage">
+        <div className="TopBarSection">
+          <Header />
+        </div>
+        <div className="MainSection">
+          <div className="SideBarSection">
+            {/* <SideNav /> */}
+          </div>
+          <div className="MainContentSection">
+            <div className="InformationBarSection">
+              <InformationBar header="Databases" showBtn={true} />
+            </div>
+            <div className="ContentSection">
+              <div
+                className={
+                  isRetrieving
+                    ? "ResourcesTable LoadingResourcesTable"
+                    : "ResourcesTable"
+                }
+              >
+                <table>
```

Comment:
i strongly discourage usage of tables, we shall need to rewrite this, and all the other `<table>` implementations

Reply:
@colinwagaba kindly check out the new changes.